portable/ThirdParty/GCC/Xtensa_ESP32/include/portmacro.h

```diff
@@ -51,11 +51,11 @@
  *  including FreeRTOS+Trace - an indispensable productivity tool, a DOS
  *  compatible FAT file system, and our tiny thread aware UDP/IP stack.
  *
- *  http://www.OpenRTOS.com - Real Time Engineers ltd license FreeRTOS to High
+ *  https://www.highintegritysystems.com/openrtos/ - Real Time Engineers ltd license FreeRTOS to High

```

Comment:
Please keep the line lengths wrapped at the same point as above.  I think it is 80 characters - and below.